Header header
float64 fAcclX
float64 fAcclY
float64 fRateZ

